app/views/posts/show.html.erb

```diff
@@ -1 +1,23 @@
-<p> here will be details about one post</p>
+<h2>details of posts<h1>
+<main>
+    <div class="post-container">
+        <h2>
+            <%= @post.title%> by <%= @post.author.name%>
+        </h2>
+        <p class="comment-like-count">
+            Comments: <%= @post.comments_counter%>, Likes:<%= @post.likes_counter%>
```

Comment:
- [ ] Kindly also fix this to match what you defined in the schema: `ComentsCounter`, `LikesCounter`.